src/Appwrite/Platform/Workers/Webhooks.php

```diff
@@ -113,7 +121,21 @@ private function execute(array $events, string $payload, Document $webhook, Docu
         }
 
         if (false === \curl_exec($ch)) {
-            $this->errors[] = \curl_error($ch) . ' in events ' . implode(', ', $events) . ' for webhook ' . $webhook->getAttribute('name');
+            $dbForConsole->increaseDocumentAttribute('webhooks', $webhook->getId(), 'attempts', 1);
+            $webhook = $dbForConsole->getDocument('webhooks', $webhook->getId());
+            $attempts = $webhook->getAttribute('attempts');
+            $logs = \curl_error($ch) . ' in events ' . implode(', ', $events);
+            $webhook->setAttribute('logs', $logs);
+            // $statusCode = curl_getinfo($ch, CURLINFO_RESPONSE_CODE);
```

Comment:
I think logs are nto useful. CURL error is often just a weird number, not indicating too much. I would do longs along those lines:

```
URL: [WEBHOOK_URL]
Method: POST
Response status code: curl_getinfo($ch, CURLINFO_RESPONSE_CODE)
CURL error code: \curl_error($ch)
Events: implode(', ', $events)
Response body:
[RESPONSE_BODY]
```

(For response body, truncate to let's say 100k characters. Take inspiration from functions or builds worker, we do it there already I think)